src/components/App/App.js

```diff
@@ -37,18 +35,26 @@ function App() {
   }
 
   useEffect(() => {
-    getIDs()
+    getIDs();
   }, [])
 
   useEffect(() => {
-    ids && getSingleArtPiece();
+    ids.length && getSingleArtPiece(shuffleItems(ids)[0]);
+    ids.length && getSingleArtPiece(shuffleItems(ids)[1]);
+    ids.length && getSingleArtPiece(shuffleItems(ids)[2]);
+    ids.length && getSingleArtPiece(shuffleItems(ids)[3]);
+    ids.length && getSingleArtPiece(shuffleItems(ids)[4]);
   }, [ids])
 
   return (
-    <div className="App">
-      {/* {ids.length && console.log('IDs: ', ids)} */}
-      {/* {wall.length && console.log('WALL: ', wall)} */}
-      {wall.length > 1 && <img src={wall[1].primaryImageSmall} />}
+    <div className="App">Salon
```

Comment:
Opportunity for refactoring; perhaps we could test one condition to render all elements.